api/controllers/AuthController.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * AuthController
+ *
+ * @description :: Server-side logic for managing auths
+ * @help        :: See http://sailsjs.org/#!/documentation/concepts/Controllers
+ */
+
+module.exports = {
```

Comment:
What else can be improved if putting your AuthController to a real production?
(You don't need to implement them, just some thoughts)

Reply:
I don't sure what kind change you want? Security or other ?

For security, the Authentication need other projection. 
- XSS, sails already provided via CSRF
- Throttling, limit the frequency of login for a same user(force password jack)
- Hash vulnerability, If user don’t existed, the don’t have a hash calculation.  It means the request response will more short than a login user existed.
- Log, wherever a user is login success or fail, we need a log to record who, when, where（ip）and what happened

for architecture,
- Token, if we practice micro-service, the AuthController need sign a  JWT or other kinds token, take over the token to the  API-gateway. share user and permission info to other service 
- Messege, broadcast a user login or logout event to the authentication topic. some other service need care about this type events. such as friend system or audit system